### PR TITLE
add symfony 4 support

### PR DESCRIPTION
**Pull request by LaetitiaRiffaud**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| BC breaks?      | yes
| Related tickets | N/A
| License         | MIT

### CODE REVIEW

**Comment by lchrusciel** [comment]
I would love to introduce the support for Symfony 4, but both of the PR's this and #101 are failing :(

**Comment by angelov** [comment]
The problem with the build is that the `app.third_party_api_client` service requested in the controller in the test application is private in Symfony 4+. @LaetitiaRiffaud I submitted the change [here](https://github.com/LaetitiaRiffaud/ApiTestCase/pull/1), please check it out.

**Comment by LaetitiaRiffaud** [comment]
@lchrusciel, thanks to @angelov tests have passed.

File: composer.json [hunk 1]
```diff
@@ -34,12 +34,12 @@
         "phpspec/php-diff": "^1.1",
         "phpunit/phpunit": "^6.0",
         "polishsymfonycommunity/symfony-mocker-container": "^1.0",
-        "symfony/browser-kit": "^3.2",
-        "symfony/finder": "^3.2",
-        "symfony/framework-bundle": "^3.2"
+        "symfony/browser-kit": "^3.2|^4.0",
+        "symfony/finder": "^3.2|^4.0",
+        "symfony/framework-bundle": "^3.2|^4.0"
```
Comment (lchrusciel) [inline composer.json]: We need to bump minimal Symfony version here as well, as `KERNEL_CLASS` has been added [in tag 3.3](https://github.com/symfony/symfony/blob/v3.3.0/src/Symfony/Bundle/FrameworkBundle/Test/KernelTestCase.php)

**Comment by lchrusciel** [comment]
Thanks a lot @angelov and @LaetitiaRiffaud! I allowed myself to make a few adjustments to your code, hope you don't mind.

**Pull request merged**